src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublicationService.cs

```diff
@@ -148,12 +148,29 @@ public class PublicationService : IPublicationService
 
                     return (await ValidatePublicationSlugUniqueForUpdate(publication.Id, updatedPublication.Slug)).Map(_ =>
                     {
+                        var statisticsDbPublication = _statisticsDbContext.Publication.Find(publication.Id);
```

Comment:
We do all the saves in the block below, so below you could just `statisticsDbPublication.Slug = publication.Slug;` before that save, rather than fetching and saving twice, and remove this bit of code.